packages/openapi-generator/src/file-serializer/operation.ts

```diff
@@ -15,7 +15,7 @@ import { serializeSchema } from './schema';
 export function serializeOperation(operation: OpenApiOperation): string {
   const requestBuilderParams = [
     `'${operation.method}'`,
-    `'${operation.pathPattern}'`
+    `"${operation.pathPattern}"`
```

Comment:
Let me check if I understood this correctly:
- there exist APIs that expect params to be passed like this: `example.com/api/entities('5')/something`
- because we used single quotes for the string, the generated code looked like this `'test('{id}')'`, which is syntactically incorrect?

Reply:
Yes, correct

Reply:
The ideas some API providers have... 😒